.config/nextest.toml

```diff
@@ -15,6 +15,13 @@ filter = 'package(~vmm_tests) and test(openhcl)'
 # Mark OpenHCL VMM tests as extra heavy, as they have to also simulate VTL2.
 threads-required = 4
 
+[[profile.default.overrides]]
+# use fuzzy-matching for the package() to allow out-of-tree tests to use the
+# same profile
+filter = 'package(~vmm_tests) and test(heavy)'
+# Mark heavy tests as extra heavy, as they include up to 16 vps.
```

Comment:
very, very minor nit: we're using extra heavy for both 4 and 16 vp tests. Maybe the 16 vp tests should be mega-heavy :-D .

But, seriously: can we introduce a lint that makes sure that test name somehow matches this profile?